docs/tutorial.md

```diff
@@ -26,26 +30,24 @@ To get a feeling for the Mineflayer and how to use it, please open a text editor
 First, create an instance of the Bot.
 
 ```js
-const mineflayer = require("mineflayer");
+const mineflayer = require('mineflayer')
 const options = {
-  host: "localhost", // optional / minecraft server address
+  host: 'localhost', // optional / minecraft server address
   port: 25565, // optional
-  username: "email@example.com", // email and password are required only for
-  password: "12345678", // online-mode=true servers
-  version: false, // false corresponds to auto version detection (that's the default), put for example '1.8.8' if you need a specific version
-};
-const bot = mineflayer.createBot(options);
+  username: 'email@example.com', // email and password are required only for
+  password: '12345678', // online-mode=true servers
+  version: false // false corresponds to auto version detection (that's the default), put for example '1.8.8' if you need a specific version
+}
+const bot = mineflayer.createBot(options)
 ```
 
 Say hi to the chat after you spawn in, using [`spawn`](http://mineflayer.prismarine.js.org/#/api?id=quotspawnquot) event and [`bot.chat()`](http://mineflayer.prismarine.js.org/#/api?id=botchatmessage) method.
 
 ```js
-// ...
-
-bot.once("spawn", function () {
-  bot.chat("hi!");
+bot.once('spawn', function () {
+  bot.chat('hi!')
   // `bot.chat()` method for sending message or commad to the chat.
```

Comment:
`bot.chat()` method **sends** message or comma**n**d to the chat.

Reply:
got it